restart_process/tilt-restart-wrapper.go

```diff
@@ -52,6 +53,18 @@ func main() {
 	cmd.Stdout = os.Stdout
 	cmd.Stderr = os.Stderr
 
+	// Set up a signal forwarding handler
+	sigs := make(chan os.Signal, 1)
+	signal.Notify(sigs)
+	go func() {
+		for {
+			sig := <-sigs
+			if cmd.Process != nil {
```

Comment:
i don't think checking cmd.Process like this is thread-safe?

we probably need something like:

```
err := cmd.Start()

// set up process control

err = cmd.Wait()
```